lib/services/presets.go

```diff
@@ -104,6 +104,10 @@ func NewSystemAutomaticAccessBotUser() types.User {
 // NewPresetEditorRole returns a new pre-defined role for cluster
 // editors who can edit cluster configuration resources.
 func NewPresetEditorRole() types.Role {
+	// IMPORTANT: Before adding new defaults, please make sure that the
+	// underlying field is supported by the standard role editor UI. This role
+	// should be editable with a rich UI, without requiring the user to dive into
+	// YAML.
```

Comment:
Can we have a test that fails when users do not follow this recommendation?

Reply:
@marcoandredinis Yes, I'm adding it in a follow-up PR (https://github.com/gravitational/teleport/pull/51608). I just wanted a stop-gap measure, because people were adding fields faster than I could add them on the UI. :)